### PR TITLE
feat: Add Node 12 community to the drop down of Node choices.

### PR DESCRIPTION
**Pull request by lholmquist**
fixes [NODE-358](https://issues.jboss.org/browse/NODE-358)

### CODE REVIEW

**Comment by gastaldi** [comment]
[test]

File: nodejs/v12-community/health-check/booster.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Node.js - Health Checks
+description: Demonstrates Health Checks in Node.js
+source:
+  git:
+    url: https://github.com/nodeshift-starters/nodejs-health-check
+    ref: master
+metadata:
+  app:
+    osio:
```
Comment (gastaldi) [inline nodejs/v12-community/health-check/booster.yaml]: The `osio`  metadata is not used anymore and can be safely removed

Comment (lholmquist) [reply nodejs/v12-community/health-check/booster.yaml]: good to know

**Comment by gastaldi** [comment]
@lholmquist feel free to merge it when it's ready

**Pull request merged**